raiden-cli/src/routes/tokens.ts

```diff
@@ -1,20 +1,52 @@
 import { Router, Request, Response } from 'express';
+import { first } from 'rxjs/operators';
+
 import { Cli } from '../types';
+import { validateAddressParameter } from '../utils/validation';
+
+async function getAllTokens(this: Cli, _request: Request, response: Response) {
+  response.json(await this.raiden.getTokenList());
+}
+
+async function getTokenNetwork(this: Cli, request: Request, response: Response) {
+  // Attention: this also enables monitoring token
+  try {
+    response.json(await this.raiden.monitorToken(request.params.tokenAddress));
+  } catch (error) {
+    response.status(404).send(error.message);
+  }
+}
+
+async function getTokenPartners(this: Cli, request: Request, response: Response) {
+  const token: string = request.params.tokenAddress;
+  const channelsDict = await this.raiden.channels$.pipe(first()).toPromise();
+  const baseUrl = request.baseUrl.replace(/\/\w+$/, '');
```

Comment:
Are you 100% sure that this works? I thought `baseUrl` just refers to the routers mounting point. So in this case `transfers`. Are you sure this works recursive and get the parent routers mount points?

Reply:
Yes, I've tested, it works as intended, it includes the whole mountpoint, including parent's. This is the only way I found besides exporting a `const` somewhere for api base url, and I prefer this way.